datahub/dbmaintenance/management/commands/update_company_great_profile_status.py

```diff
@@ -0,0 +1,51 @@
+from logging import getLogger
+
+import reversion
+
+from datahub.company.models import Company
+from datahub.dbmaintenance.management.base import CSVBaseCommand
+from datahub.dbmaintenance.utils import parse_bool, parse_uuid
+from datahub.search.signals import disable_search_signal_receivers
+
+
+logger = getLogger(__name__)
+
+
+class Command(CSVBaseCommand):
+    """Command to update Company.great_profile_status."""
+
+    @disable_search_signal_receivers(Company)
+    def _handle(self, *args, **options):
+        """
+        Disables search signal receivers for companies.
+        This is to avoid generating huge numbers of sync thread-pool tasks for companies updating.
```

Comment:
For info, the thread pool is something we were using before Celery (unfortunately the other comment was out of date), so it would be good to update this one e.g.:

```suggestion
        This is to avoid queuing huge numbers of sync Celery tasks for companies updating.
```